refresh-data.sh

```diff
@@ -69,9 +69,7 @@ refresh_data() {
 }
 
 update_index() {
-    echo 'Updating search indexes'
-    ./cfgov/manage.py opensearch index --force rebuild
-    ./cfgov/manage.py opensearch document --force --refresh --parallel index
+  source ./index.sh
```

Comment:
Since index.sh is chmoded+x and has a `#!/bin/bash`, does this need to be sourced, or should it just be called?

Reply:
I `source`d it so that it would run in the same process, which maybe is unnecessary, but seemed reasonable